### DISCUSSION TITLE
faq

### DISCUSSION THREAD

**Discussion by ghost** [PowerShell/PowerShell#21516] [Q&A] [answered]

**Comment by rhubarb-geek-nz** [comment, marked as answer]
> but specific to powershell?

What features are you expecting that make it specific to PowerShell? Any file storage or source code system can store PowerShell scripts. Most web-based source code systems will do syntax highlighting for PowerShell scripts.

The only specific PowerShell site I know of is the [PowerShell Gallery](https://www.powershellgallery.com) which is for hosting finished public modules, more like an app store.
